pom.xml

```diff
@@ -28,6 +28,12 @@
             <version>${mockito-all.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <!--MySQL Dependencies-->
+            <groupId>mysql</groupId>
+            <artifactId>mysql-connector-java</artifactId>
+            <version>5.1.47</version>
```

Comment:
You need to create a property within the `<properties>`section.

Reply:
Not sure which one is needed

Reply:
I meant that you can, instead of using `<version>5.1.47</version>`, create a property within the properties section in the pom.xml file as follow:

```
<properties>
        <junit.version>4.11</junit.version>
        <mockito-all.version>1.10.19</mockito-all.version>
        <mysql-connector-java.version>5.1.47</mysql-connector-java.version>
</properties>
```

And then use it as follow:

```
<version>${mysql-connector-java.version}</version>
```

Reply:
ok